src/components/WorkOrder/Photos/PhotosTab.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types'
+import { useState, useEffect } from 'react'
+import Spinner from '../../Spinner'
+import { frontEndApiRequest } from '@/utils/frontEndApiClient/requests'
+import ErrorMessage from '../../Errors/ErrorMessage'
+import UploadPhotosForm from './UploadPhotosForm'
+import PhotoViewList from './PhotoViewList'
+
+const PhotosTab = ({ workOrderReference }) => {
+  const [photos, setPhotos] = useState([])
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState()
+
+  const getPhotos = async (workOrderReference) => {
+    setError(null)
```

Comment:
Is this just different aesthetics, or different behaviour?

I'm wonder why isn't it
```js
const [error, setError] = useState(null)
```

Instead of:
```js
const [error, setError] = useState()
//...
setError(null)
```
As far as I see, based on `useEffect` this only gets re-rendered during the initial load.

Reply:
undefined and null are both falsy, so it doesnt make a difference. But I can set the default value to null for clarity.

`getPhotos` is also called when files have been uploaded. That line is there to clear any previous error message